google/auth/pluggable.py

```diff
@@ -171,9 +198,10 @@ def retrieve_subject_token(self, request):
         env = os.environ.copy()
         env["GOOGLE_EXTERNAL_ACCOUNT_AUDIENCE"] = self._audience
         env["GOOGLE_EXTERNAL_ACCOUNT_TOKEN_TYPE"] = self._subject_token_type
-        env[
-            "GOOGLE_EXTERNAL_ACCOUNT_INTERACTIVE"
-        ] = "0"  # Always set to 0 until interactive mode is implemented.
+        env["GOOGLE_EXTERNAL_ACCOUNT_INTERACTIVE"] = (
```

Comment:
Implementation is missing GOOGLE_EXTERNAL_ACCOUNT_ID and GOOGLE_EXTERNAL_ACCOUNT_REVOKE